backend/server-api/src/test/java/org/sonarsource/sonarlint/core/serverapi/projectbindings/ProjectBindingsApiTests.java

```diff
@@ -46,49 +46,97 @@ void setUp() {
   }
 
   @Test
-  void should_return_project_id_by_url() {
+  void should_return_project_id_by_url_for_sqc() {
     var url = "https://github.com/foo/bar";
     var encodedUrl = URLEncoder.encode(url, StandardCharsets.UTF_8);
     mockServer.addStringResponse("/dop-translation/project-bindings?url=" + encodedUrl,
       "{\"bindings\":[{\"projectId\":\"proj:123\"}]}");
 
-    var result = underTest.getProjectBindings(url, new SonarLintCancelMonitor());
+    var result = underTest.getSQCProjectBindings(url, new SonarLintCancelMonitor());
 
-    assertThat(result).isEqualTo(new ProjectBindingsResponse("proj:123"));
+    assertThat(result).isEqualTo(new SQCProjectBindingsResponse("proj:123"));
   }
 
   @Test
-  void should_return_empty_when_no_bindings() {
+  void should_return_empty_when_no_bindings_for_sqc() {
     var url = "https://github.com/foo/bar";
     var encodedUrl = URLEncoder.encode(url, StandardCharsets.UTF_8);
     mockServer.addStringResponse("/dop-translation/project-bindings?url=" + encodedUrl,
       "{\"bindings\":[]}");
 
-    var result = underTest.getProjectBindings(url, new SonarLintCancelMonitor());
+    var result = underTest.getSQCProjectBindings(url, new SonarLintCancelMonitor());
 
     assertThat(result).isNull();
   }
 
   @Test
-  void should_return_empty_when_invalid_json() {
+  void should_return_empty_when_invalid_json_for_sqc() {
     var url = "https://github.com/foo/bar";
     var encodedUrl = URLEncoder.encode(url, StandardCharsets.UTF_8);
     mockServer.addStringResponse("/dop-translation/project-bindings?url=" + encodedUrl,
       "this is not json");
 
-    var result = underTest.getProjectBindings(url, new SonarLintCancelMonitor());
+    var result = underTest.getSQCProjectBindings(url, new SonarLintCancelMonitor());
 
     assertThat(result).isNull();
   }
 
   @Test
-  void should_return_empty_when_request_fails() {
+  void should_return_empty_when_request_fails_for_sqc() {
```

Comment:
Random idea: Instead of suffixing every method, you could also use `@Nested` and have one for SQS and one for SQC